docker-compose.yaml

```diff
@@ -13,8 +13,8 @@ services:
       target: develop
```

Comment:
These changes should be removed, they are style-only and unrelated to the ticket